### PR TITLE
STS-43 validator bonding

### PR DESCRIPTION
**Pull request by momipsl**
- Renamed commands:
`nctl-log-view -> nctl-view-node-log`

- New commands:
`nctl-view-node-storage`  
`nctl-wg-200`  
`nctl-wg-201`  
`nctl-wg-210`  
`nctl-wg-211`  

- Command synonyms:  
`nctl-down  ==  nctl-stop`  
`nctl-up  ==  nctl-start`  
`nctl-do-transfer == nctl-wg-100`  
`nctl-do-transfer-wasm == nctl-wg-110`  
`nctl-do-auction-submit == nctl-wg-200`  
`nctl-do-auction-withdraw == nctl-wg-201`  
`nctl-do-auction-delegate == nctl-wg-210`  
`nctl-do-auction-undelegate == nctl-wg-211`

- Command behaviour changes
`
nctl-compile-client - compiles all client side contracts as well as client itself
`

### CODE REVIEW

**Review by linclelinkpart5 — COMMENTED** [review]
Makes sense, I appreciate the aliasing, and the new name for `nctl-view-node-log`!

**Review by Fraser999 — CHANGES_REQUESTED** [review]
A few issues:

* For `nctl-wg-210` I get `nctl-wg-210: command not found`.
* For `nctl-do-auction-delegate` I get a panic with `can't parse  as PublicKey: AsymmetricKey("too short")`
* For `nctl-wg-211` I get a panic with `can't parse  as PublicKey: AsymmetricKey("too short")`
* For `nctl-do-auction-undelegate` I get `bash: /home/fraser/Rust/nctl/sh/generators/wg_212.sh: No such file or directory`

File: docs/commands.md [hunk 1]
```diff
@@ -136,42 +135,132 @@ e.g. `nctl-stop net=1 node=all`  (same as above)
...
+NOTE - this command has a synonym: `nctl-do-transfer`
 
 ### nctl-wg-110 net={X:-1} node={Y:-1} payment={P:-200000} gas={G:-10} transfers={T:-100} interval={I:-0.01} user={U:-1}
 
 Dispatches to node Y in network X, T wasm based transfers from network faucet to user #U.  If node=all then transfers are dispatched to nodes in a round-robin fashion.
 
 e.g. `nctl-wg-110`
 
-e.g. `nctl-wg-110 net=1 node=1 payment=200000 gas=10 transfers 100 interval=0.01 user=1`  (same as above)
+e.g. `nctl-wg-110 net=1 node=1 payment=200000 gas=10 transfers100 interval=0.01 user=1`  (same as above)
```
[the remark this answers is not in the document]
Comment (momipsl) [reply docs/commands.md]: Fixed

File: docs/commands.md [hunk 2]
```diff
@@ -136,42 +135,132 @@ e.g. `nctl-stop net=1 node=all`  (same as above)
...
+
+e.g. `nctl-wg-110 transfers=10000 interval=0.001`
+
+NOTE - this command has a synonym: `nctl-do-transfer-wasm`
+
+### nctl-wg-200 net={X:-1} node={Y:-1} amount={A:-1000000} rate={R:-125} payment={P:-200000} gas={G:-10} user={U:-1}
+
+Dispatches to node Y in network X from user #U, a Proof-Of-Stake auction bid **submission** for amount amount A (motes) with a delegation rate of R.  Displays relevant deploy hash for subsequent querying.
```
[the remark this answers is not in the document]
Comment (momipsl) [reply docs/commands.md]: Fixed

File: docs/commands.md [hunk 3]
```diff
@@ -136,42 +135,132 @@ e.g. `nctl-stop net=1 node=all`  (same as above)
...
+
+e.g. `nctl-wg-200 net=1 node=1 amount=1000000 rate=125 payment=200000 gas=10 user=1`  (same as above)
+
+e.g. `nctl-wg-200 amount=2000000 rate=250`
+
+NOTE - this command has a synonym: `nctl-do-auction-submit`
+
+### nctl-wg-201 net={X:-1} node={Y:-1} amount={A:-1000000} payment={P:-200000} gas={G:-10} user={U:-1}
+
+Dispatches to node Y in network X from user #U, a Proof-Of-Stake auction bid **withdrawal** for amount amount A (motes).  Displays relevant deploy hash for subsequent querying.
```
[the remark this answers is not in the document]
Comment (momipsl) [reply docs/commands.md]: Fixed

File: docs/commands.md [hunk 4]
```diff
@@ -136,42 +135,132 @@ e.g. `nctl-stop net=1 node=all`  (same as above)
...
+
+e.g. `nctl-wg-201 net=1 node=1 amount=1000000 payment=200000 gas=10 user=1`  (same as above)
+
+e.g. `nctl-wg-201 amount=2000000`
+
+NOTE - this command has a synonym: `nctl-do-auction-withdraw`
+
+### nctl-wg-210 net={X:-1} node={Y:-1} amount={A:-1000000} payment={P:-200000} gas={G:-10} user={U:-1}
+
+Dispatches to node Y in network X from user #U, a Proof-Of-Stake **delegate** bid for amount amount A (motes).  Displays relevant deploy hash for subsequent querying.
```
[the remark this answers is not in the document]
Comment (momipsl) [reply docs/commands.md]: Fixed

File: docs/commands.md [hunk 5]
```diff
@@ -136,42 +135,132 @@ e.g. `nctl-stop net=1 node=all`  (same as above)
...
+
+e.g. `nctl-wg-210 net=1 node=1 amount=1000000 payment=200000 gas=10 user=1`  (same as above)
+
+e.g. `nctl-wg-210 amount=2000000`
+
+NOTE - this command has a synonym: `nctl-do-auction-delegate`
+
+### nctl-wg-211 net={X:-1} node={Y:-1} amount={A:-1000000} payment={P:-200000} gas={G:-10} user={U:-1}
+
+Dispatches to node Y in network X from user #U, a Proof-Of-Stake **undelegate** bid for amount amount A (motes).  Displays relevant deploy hash for subsequent querying.
```
[the remark this answers is not in the document]
Comment (momipsl) [reply docs/commands.md]: Fixed

File: docs/installation.md [hunk 6]
```diff
@@ -8,7 +8,12 @@
 ### Step 1 - install pre-requisites.
 
 ```
+# Supervisor - cross-platform process manager.
 python3 -m pip install supervisor
+
+# Rust toolchain - required by casper-node software.
+cd YOUR_WORKING_DIRECTORY/casper-node
+make make setup-rs
```
[the remark this answers is not in the document]
Comment (momipsl) [reply docs/installation.md]: Fixed

**Pull request merged**